docs/new-architecture-app-intro.md

```diff
@@ -27,7 +27,59 @@ Starting from React Native `0.69.0`, you may also need to update the version of
 yarn add react@18.0.0
 ```
```

Comment:
General feedback here: those instructions are misleading.

IMHO:  To update to the most recent version of React Native, you can run this command:

```
npx react-native upgrade
```

By doing `yarn add react-native@0.70.0` we expose users to misalignment of the React version.

Specifically 0.70 is on React 18.1.0: https://github.com/facebook/react-native/blob/0b4b7774e2d71259962ed36b7acb5c3989c3be9c/package.json#L97

0.71 will most likely be on 18.2.0 and so on..

Reply:
Will `npx react-native upgrade` upgrade also its dependencies?